posthog/tasks/verify_persons_data_in_sync.py

```diff
@@ -0,0 +1,147 @@
+import json
+from collections import Counter, defaultdict
+from datetime import timedelta
+from typing import Any, Dict, List
+
+import structlog
+from django.db.models.query import Prefetch
+from django.utils.timezone import now
+
+from posthog.celery import app
+from posthog.client import sync_execute
+from posthog.models.person import Person
+
+logger = structlog.get_logger(__name__)
+
+# We check up to LIMIT persons between PERIOD_START..PERIOD_END, in batches of BATCH_SIZE
+# This helps keep the metric "moving" as we ship fixes or bugs.
+LIMIT = 100000
```

Comment:
Team Infra has asked us to not fetch more than 10k rows from Postgres at a time - I think that's valid and we should do a paginated fetch here

Reply:
We're fetching only IDs here - tested, this query takes ~50ms and isn't that much data. Loading properties is much heavier.